src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let statedCopy = { ...state };
```

Comment:
```suggestion
    const stateCopy = { ...state };
```
it is better to use const here, object is not primitive, you can change it's properties if it will be const
stated -> state